.github/workflows/ci.yml

```diff
@@ -31,21 +31,25 @@ jobs:
           - macOS-latest
     steps:
       - uses: actions/checkout@master
+      - run: git submodule update --init --recursive
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm run coverage
+      - run: npm run wpt -- --summary-only
 
   test-nightly:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
-      - name: npm run coverage
+      - run: git submodule update --init --recursive
+      - name: coverage + wpt
         run: |
           curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
           export NVM_DIR=~/.nvm
           source ~/.nvm/nvm.sh
           NVM_NODEJS_ORG_MIRROR=https://nodejs.org/download/nightly/ nvm install node
-          npm install
-          npm run coverage
+          npm install && \
+          npm run coverage && \
+          npm run wpt -- --summary-only
```

Comment:
Is there a reason to concatenate the commands? Will CI not fail if one command fails?

Reply:
Not in this case, this whole step includes installing nvm, nightly node and running coverage and wpt, only if wpt failed would the whole step fail.

Reply:
the whole step is to github ci one observable command, so if coverage failed but wpt succeeded, so would the step. I didn't feel like spending cycles on splitting installing nvm, installing nightly and then loading nvm to shell again for each of coverage / wpt...

Reply:
might as well remove the nightly altogether tho

Reply:
Doesn't `&&` just mean "fail early"?

Reply:
CI would only look at the exit code of the last command ran... i've removed nightly tho, so this is no a issue.

Reply:
if you `set -e` then it will look at every exit code.

Reply:
@panva Unless GitHub actions changes the default shell behavior, `&&` will fail early. For example, `rm does-not-exist && echo yes` will not `echo yes`, because the first command fails. So wpt would not have been run, if coverage had failed.